corehq/apps/reports/standard/users/reports.py

```diff
@@ -133,6 +134,20 @@ def _user_history_row(record, timezone):
 
 
 def _details_cell(changes):
+    def _html_list(changes, unstyled=True):
```

Comment:
I'm going to say the current code is more accurate/specific, given that the class in question is called "unstyled".

Reply:
sounds reasonable.

I was just thinking of it as adding proper tags to make it "pretty" in html.
Its just personal preference to avoid negative bool (if that is the right way to refer this)